packages/flutter_tools/lib/src/ios/ios_deploy.dart

```diff
@@ -348,6 +348,12 @@ class IOSDeployDebugger {
           .transform<String>(utf8.decoder)
           .transform<String>(const LineSplitter())
           .listen((String line) {
+
+        // TODO(vashworth): Revert after https://github.com/flutter/flutter/issues/121231 is resolved.
+        if (line.isNotEmpty) {
+          _logger.printTrace(line);
```

Comment:
Can you add a test for this in test/general.shard/ios/ios_deploy_test.dart? Looks like we have prior art.

Reply:
My bad, I thought just logs didn't need a test. Also, not sure what you mean by "prior art", but I added a test.